### ISSUE TITLE
Add more locations

### ISSUE THREAD

**Issue by carysmills** [carysmills/whentoshop#27]
## Ottawa and nearby

 - [ ] osgoode

- [ ] kemptville

## Outside of Ottawa

- [ ] Junction/High Park or maybe Junction/Stockyards

- [ ] Lower Hamilton, and other suggest maybe the Mountain 

- [ ] Metro in Toronto on Front st

## Outside of Ontario

- [ ] Independent and Nester's in Squamish BC

- [ ] North Vancouver
Loblaw city market

- [ ] Independent Grocery store in Squamish, BC

- [ ]  St. John’s : 
Costco on Danny Drive
Dominion at 55 Stavanger Drive 
Walmart Superstore at 90 Aberdeen Ave.
*As of May 9, no data is being brought back

**Comment by EricStG** [comment]
Would you accept a PR for Aylmer/Hull/Gatineau?

**Comment by carysmills** [comment]
@EricStG yes definitely! Let me know if you have any issues getting things working!
